autosklearn/pipeline/components/classification/gradient_boosting.py

```diff
@@ -125,34 +93,25 @@ def get_properties(dataset_properties=None):
     @staticmethod
     def get_hyperparameter_search_space(dataset_properties=None):
         cs = ConfigurationSpace()
-        loss = Constant("loss", "deviance")
+        loss = Constant("loss", "auto")
         learning_rate = UniformFloatHyperparameter(
             name="learning_rate", lower=0.01, upper=1, default_value=0.1, log=True)
-        n_estimators = UniformIntegerHyperparameter(
-            "n_estimators", 50, 500, default_value=100)
         max_depth = UniformIntegerHyperparameter(
-            name="max_depth", lower=1, upper=10, default_value=3)
-        criterion = CategoricalHyperparameter(
-            'criterion', ['friedman_mse', 'mse', 'mae'],
-            default_value='mse')
-        min_samples_split = UniformIntegerHyperparameter(
-            name="min_samples_split", lower=2, upper=20, default_value=2)
+            name="max_depth", lower=2, upper=10, default_value=3)
         min_samples_leaf = UniformIntegerHyperparameter(
             name="min_samples_leaf", lower=1, upper=20, default_value=1)
-        min_weight_fraction_leaf = UnParametrizedHyperparameter("min_weight_fraction_leaf", 0.)
-        subsample = UniformFloatHyperparameter(
-                name="subsample", lower=0.01, upper=1.0, default_value=1.0)
-        max_features = UniformFloatHyperparameter(
-            "max_features", 0.1, 1.0 , default_value=1)
         max_leaf_nodes = UnParametrizedHyperparameter(
             name="max_leaf_nodes", value="None")
-        min_impurity_decrease = UnParametrizedHyperparameter(
-            name='min_impurity_decrease', value=0.0)
-        cs.add_hyperparameters([loss, learning_rate, n_estimators, max_depth,
-                                criterion, min_samples_split, min_samples_leaf,
-                                min_weight_fraction_leaf, subsample,
-                                max_features, max_leaf_nodes,
-                                min_impurity_decrease])
+        max_iter = UniformIntegerHyperparameter(
+            "max_iter", 50, 500, default_value=100)
+        max_bins = UniformIntegerHyperparameter(
+            name="max_bins", lower=2, upper=256, default_value=256, log=False)
```

Comment:
I think we shouldn't tune this value as it mostly covers the training speed which is not an optimization objective of Auto-sklearn.

Reply:
ok